modules/issue_tracker/php/issuerowprovisioner.class.inc

```diff
@@ -99,7 +99,22 @@ class IssueRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
             $module = &$modules[$mname];
         }
 
-        $cids = isset($row['centerId']) ? [$row['centerId']] : $this->_siteList;
+        if (isset($row['centerId'])) {
+            $cids = [intval($row['centerId'])];
+        } else {
+            $cids = $this->_siteList;
+        }
+
+        if (isset($row['centerId'])) {
+            $row['centerId'] = [$row['centerId']];
+        } else {
+            $row['centerId'] = array_map(
+                "strval",
+                \User::singleton()->getCenterIDs()
+            );
+            array_push($row['centerId'], "all");
+        }
```

Comment:
I don't understand what you're trying to do here.

Reply:
@driusan I added comments to clarify this section.